src/main/java/org/prebid/server/bidder/smaato/SmaatoBidder.java

```diff
@@ -162,37 +166,37 @@ private Site modifySite(Site site, String firstPublisherId) {
     }
 
     private User modifyUser(User user) {
-        final ExtUser ext = user.getExt();
-        if (ext == null) {
-            return user;
+        if (user == null) {
+            return null;
         }
 
-        final ObjectNode extDataNode = ext.getData();
-        if (extDataNode == null) {
+        final ExtUser userExt = user.getExt();
+        final ObjectNode extDataNode = userExt != null ? userExt.getData() : null;
+        if (extDataNode == null || extDataNode.isNull()) {
```

Comment:
As we have ObjectNode it has method 
`public JsonNodeType getNodeType() {
        return JsonNodeType.OBJECT;
    }`
so method isNull will always return false, please use `isEmpty()` method

Reply:
Nice catch